src/bp-templates/bp-nouveau/includes/activity/template-tags.php

```diff
@@ -1944,6 +1944,40 @@ function bb_nouveau_get_activity_entry_bubble_buttons( $args ) {
 		);
 	}
 
+	$notification_type = bb_activity_enabled_notification( 'bb_activity_comment', bp_loggedin_user_id() );
+	if ( ! empty( $notification_type ) && ! empty( array_filter( $notification_type ) ) ) {
+
+		$unmute_action_class        = 'bb-icon-bell-slash';
+		$unmute_action_label        = __( 'Turn Off Notification', 'buddyboss' );
+		$activity_mute_notification = bp_activity_get_meta( $activity_id, 'muted_notification_users' );
```

Comment:
@kishanbuddyboss Use this function `bb_user_has_mute_notification`

Reply:
Fixed it